### PR TITLE
Add responsive size

### PR DESCRIPTION
**Pull request by Tatayoyoh**
Hi @dbetka !
I would like to contribute to your great package by adding responsive size like '2.5em' or '250%'.

Value can be passed as app project configuration
```
import materialSymbolsPlugin from '@dbetka/vue-material-symbols';
import 'material-symbols/index.css';

const app = createApp(App);
app.use(materialSymbolsPlugin, {
  defaultSize: '1.5em',
});
```

or attribute
```
<MaterialSymbol name="delete" size="1.5em">
<MaterialSymbol name="delete" size="150%">
```

Can you validate this evolution and release a new package version if ok ?
Thanks a lot for your code :)

### CODE REVIEW

**Comment by dbetka** [comment]
Hi @Tatayoyoh,
Thanks for your commitment!

This is a great idea and I will look at it tonight or tomorrow morning. 🙂

File: src/MaterialSymbol.vue [hunk 1]
```diff
@@ -31,11 +33,12 @@ const styles = computed(() => {
   const fill = props.filled ? '1' : '0'
   const weight = props.weight
   const grade = GRADES[props.grade]
-  const size = props.size
+  const size = Number.isIntefer(props.size) ? `${props.size}px` : props.size
+  const opsz = Number.isIntefer(props.size) ? props.size : window.$materialSymbolsDefaults?.defaultSize || DEFAULT_SIZE,
```
Comment (dbetka) [inline src/MaterialSymbol.vue]:
```suggestion
  const size = Number.isInteger(props.size) ? `${props.size}px` : props.size
  const opsz = Number.isInteger(props.size) ? props.size : window.$materialSymbolsDefaults?.defaultSize || DEFAULT_SIZE
```

**Review by dbetka — CHANGES_REQUESTED** [review]
@Tatayoyoh Idea looks good and it seems to work after small corrections.
I think we should add some relative value to `demo/DemoComponent.vue`.

For example change this
```js
const sizes = [ undefined, 40, 56, 72, 96, 128 ]
```
to this
```js
const sizes = [ undefined, 40, 56, 72, 96, '7.5rem' ]
```

Are you able to make this modifications? 
After that I will release a new package version.

**Comment by Tatayoyoh** [comment]
Hey, I fixed two ugly code errors from my previous pull request and added the example into `demo/DemoComponent.vue` like you said. I can see conflicts now, sorry I have not seen you made changes :open_mouth:

File: demo/DemoComponent.vue [hunk 2]
```diff
@@ -1,10 +1,8 @@
 <script setup lang="ts">
-import { MaterialSymbol, useSymbols } from '@root'
 
-const sizes = [ undefined, 40, 56, 72, 96, 128 ]
+import { MaterialSymbol } from '@'
 
-const { metadata } = useSymbols()
-console.log(metadata)
+const sizes = [ undefined, 40, 56, 72, 96, "7.5em" ]
```
Comment (dbetka) [inline demo/DemoComponent.vue:5]: Single quote is preferable by eslint but I will change it right before release.

**Review by dbetka — APPROVED** [review]
PR is ready to merge and release today - Thank you for your commitment :)

**Comment by dbetka** [comment]
@Tatayoyoh new version has been released 🚀 
Check if everything works fine

Demo on Github page works fine https://dbetka.github.io/vue-material-symbols/

**Comment by Tatayoyoh** [comment]
Thank you a lot for this ! :1st_place_medal:

**Pull request merged**